apps/hash-graph/libs/graph/src/ontology/domain_validator.rs

```diff
@@ -144,10 +144,10 @@ impl ValidateOntologyType<PropertyType> for DomainValidator {
             });
         };
 
-        // TODO - check that the user has write access to the shortname, this will require us
+        // TODO: check that the user has write access to the shortname, this will require us
         //  making the graph aware of shortnames. We can store them alongside accountIds. We should
         //  not have to make the graph aware of User entities being a thing however.
-        //  https://app.asana.com/0/1202805690238892/1202931031833224/f
+        //  See https://linear.app/hash/issue/H-3010
```

Comment:
```suggestion
        // TODO: check that the user has write access to the shortname, this will require us
        //       making the graph aware of shortnames. We can store them alongside accountIds. We should
        //       not have to make the graph aware of User entities being a thing however.
        //  see https://linear.app/hash/issue/H-3010
```